### PR TITLE
feat: added card component

### PR DESCRIPTION
**Pull request by ShekharDhangar**
Added Card Component. Please refer deploy link or screenshots for visuals.

image

image

image

### CODE REVIEW

File: components/Cards/card.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+                <a class="link flex btn-xs" href="#">Design</a>
+            </div>
+            <div class="card-content">
+                <h1 class="title">Accessibility tools for designers and developers</h1>
+                <p class="card-description" >Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quia nostrum quis blanditiis at excepturi veritatis laborum cupiditate et aliquid molestiae?</p>
+            </div>
+            <div class="card-footer">
+                <button class="btn btn-sm">Read More</button>
+                <div class="card-author-box flex">
+                    <img class="avatar-circle size-sm"  src="https://learnnode.com/images/square-small.jpg" alt="">
```
Comment (vrx29) [inline components/Cards/card.html]: fill alt tag will some data dont leave it blank

File: components/Cards/card.css [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+    margin: 0;
+}
+.vertical-card .card-content 
+.card-description .title{
+    font-size: 1.75rem;
+    font-weight: bold;
+}
+.card-content .rating{
+    font-size: 1.3rem;
+    font-weight: 400;
```
Comment (anik31) [inline components/Cards/card.css]: Don't use font-weight as bold, bolder & 400. Use font-weight as 400,500,etc.. as imported from google fonts consistently.

**Comment by ShekharDhangar** [comment]
I have made the changes. Please review it.

**Review by anik31 — APPROVED** [review]
Looks good

**Comment by yash9263** [comment]
looks good to me

**Pull request merged**